### PR TITLE
Ab support

### PR DESCRIPTION
**Pull request by abushev**
1) Added ab support
2) Added parse json errors handling

### CODE REVIEW

**Comment by abushev** [comment]
Have no idea how to make pull request correctly :)

File: CRUD/service/connection.hpp [hunk 1]
```diff
@@ -77,7 +77,7 @@ class connection
                 request_, buffer_.data(), buffer_.data() + bytes_transferred);
             auto itr = std::find_if( request_.headers.begin(), 
                                      request_.headers.end(), 
-                                     [](const header &h) { return h.name == "Content-Length" ; }
+                                     [](const header &h) { return h.name == "content-length" ; }
             ) ;
             if ( itr != request_.headers.end()) {
                 request_.data = std::string(data, boost::lexical_cast<long>(itr->value)) ;
```
Comment (venediktov) [inline CRUD/service/connection.hpp]: I normally change  CRUD directly here 
https://github.com/venediktov/CRUD 
then issue following command in vanilla-rtb 
git subtree pull --prefix CRUD git@github.com:venediktov/CRUD.git master --squash

So if you do that , we should get it changed in the external source , then merged into vanilla-rtb 
Let me know if  you can do it , if not I will accept your merge and later change the original repository.

Comment (abushev) [inline CRUD/service/connection.hpp]: Ok, so I should make pull request into CRUD now, right?

Comment (venediktov) [inline CRUD/service/connection.hpp]: yes

Comment (venediktov) [inline CRUD/service/connection.hpp]: ok , I will merge your changes from CRUD into vanilla-rtb directly , I think it's cleaner that way and then if no conflicts , then I will accept your pending pull request from vanilla-rtb , so let's see how it works.
Thank you , Arseny !

**Comment by venediktov** [comment]
Nice work Arseny  !  looking forward to working with you!

**Pull request merged**